src/index.html

```diff
@@ -17,6 +17,87 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="container">
+        <a
+          href="#"
+          class="logo"
+        >
+          <img
+            src="./images/logo.png"
+            alt="logo"
+          />
+        </a>
+        <nav class="nav">
+          <ul class="menu">
+            <li class="menu-item">
+              <a
+                class="menu-link is-active"
```

Comment:
Fix classes everywhere
```suggestion
          <ul class="nav__list">
            <li class="nav__item">
              <a
                class="nav__link is-active"
```